src/main/resources/org/sagebionetworks/web/client/widget/entity/tabs/DiscussionTabViewImpl.ui.xml

```diff
@@ -3,9 +3,15 @@
 	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
 	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
-	<b:Row ui:field="mainContainer" >
+	<b:Row>
+		<bh:Div addStyleNames="margin-left-15 margin-right-15">
+			<g:Button ui:field="newThreadButton" styleName="btn btn-lg btn-primary margin-top-10 right" text="New Thread"/>
```

Comment:
change to 
<b:Button ui:field="primaryButton" type="PRIMARY">New Thread</b:Button>